DDR4 OC Guide.md

```diff
@@ -420,6 +420,8 @@ Some terminology:
   | 1000 | 65 - 75 |
   | 2000 | 60 - 70 |
   | 3000 | 65 - 75 (1:1 MCLK:FCLK) <br/> 75+ (2:1 MCLK:FCLK) |
+  | 4000/5000G | ??? |
```

Comment:
```suggestion
  | 4000/5000G | 55 - 65 |
```

IIRC this is 55 - 65 ns though with high enough FCLK and B-die you can definitely do 50ns